CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Next Version]
 
+### New features and optimizations
+
+### Breaking changes
+
+### Bug Fixes
+- Fixed issue where random sampler keeps state when resetting for validation, leading to a different validation batch each validation step. Fixed by using a deterministic sampler
```

Comment:
@odelalleau also fixed this for comment with the random sampler fixes

Reply:
Ok for me -- I may rewrite it when I implement the version we discussed.